src/client/js/dialog.js

```diff
@@ -29,9 +29,11 @@ function handleEvent (e) {
     case e.target.matches('a[href*="dialog/"]'):
       e.preventDefault()
       openDialog(e.target.href)
+      window.gtag('event', 'Dialog', { action: 'open', page_location: location.href })
       break
     case e.target.matches('button[data-dialog]'):
       openDialog(`dialog/${e.target.dataset.dialog}`)
+      window.gtag('event', 'Dialog', { action: 'open', page_location: location.href })
```

Comment:
I think both these gtag calls are now covered by the one in `openDialog()`?

```suggestion
      break
    case e.target.matches('button[data-dialog]'):
      openDialog(`dialog/${e.target.dataset.dialog}`)
```